src/frontend/src/handler/create_source.rs

```diff
@@ -317,6 +317,10 @@ pub(crate) async fn bind_columns_from_source(
         ..Default::default()
     };
 
+    if source_schema.format == Format::Debezium {
+        try_consume_string_from_options(&mut format_encode_options_to_consume, DEBEZIUM_IGNORE_KEY);
+    }
```

Comment:
@xiangjinwu I am not sure where to put this code, ignore_key seems the first format accepting options.